pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -117,6 +117,10 @@ func newCertRotationController(
 		rotationDay = day
 		klog.Warningf("!!! UNSUPPORTED VALUE SET !!!")
 		klog.Warningf("Certificate rotation base set to %q", rotationDay)
+	} else {
+		// for the development cycle, make the rotation 60 times faster (every twelve hours or so).
+		// This must be reverted before we ship
```

Comment:
Can we design it to have a way to set in install-config.yaml to pass the validity duration?

Reply:
Agree with you, reducing the code change.

Reply:
We don't want to see this accessible in any way to customers without tainting the cluster as unsupported.